app/models/concerns/applicable_to_nations.rb

```diff
@@ -0,0 +1,30 @@
+module ApplicableToNations
+  extend ActiveSupport::Concern
+
+  included do
+    validate :applicable_to_at_least_one_nation
```

Comment:
Just that they look like an actual URL. For example, whitehall uses a [validator for URIs](https://github.com/alphagov/whitehall/blob/master/app/validators/uri_validator.rb)